### PR TITLE
Add NFS stateful "move" migration scenario

### PR DESCRIPTION
**Pull request by djwhatle**
Documents a migration scenario involving movement of NFS PVs between two clusters that have shared access to a central NFS server.

### CODE REVIEW

File: docs/scenarios/nfs-pv/README.md [hunk 1]
```diff
@@ -9,5 +9,320 @@ This scenario walks through Migration of a stateful OpenShift app with Persisten
...
+spec:
+  # [!] Change namespaces to adjust which OpenShift namespaces should be migrated 
+  #     from source to destination cluster.
+  namespaces:
+  - mysql-persistent
+
+[... snipped, see config/samples/mig-plan.yaml for other required fields ...]
+```
+
+We also need to create a MigMigration with `quiescePods: true` since only one MySQL Pod can hold the lock at once, and we this scenario covers the 'move' PV action that re-mounts the *source cluster* NFS PVs on the *destination cluster*
```
Comment (dymurray) [inline docs/scenarios/nfs-pv/README.md]: `and we this scenario` -> `and this scenario`

File: docs/scenarios/nfs-pv/README.md [hunk 2]
```diff
@@ -9,5 +9,320 @@ This scenario walks through Migration of a stateful OpenShift app with Persisten
...
+  Migration Completed:     true
+  Start Timestamp:         2019-05-31T14:22:08Z
+  Task Phase:              Completed
+```
+
+---
+
+### 7. Inspect migration results on destination cluster
+
+Logging into the destination cluster, we should now see a running mysql pod in the 'mysql-persistent' namespace, matching what existed on the source cluster.
```
Comment (dymurray) [inline docs/scenarios/nfs-pv/README.md]: NIT: put mysql-persistent in backticks: `mysql-persistent`

Comment (djwhatle) [reply docs/scenarios/nfs-pv/README.md]: Thanks, I meant to put all of the mentions of `mysql-persistent` in backticks for their first mention in each paragraph, missed this one.

File: docs/scenarios/nfs-pv/README.md (hunk 1 above)
Comment (djwhatle) [reply docs/scenarios/nfs-pv/README.md]: +1 I noticed I did this but lost track of it.

**Pull request merged**